client/src/app/barmen/list/barmen-list.component.ts

```diff
@@ -13,6 +13,9 @@ export class BarmenListComponent implements OnInit {
 
   displayedColumns = ['nickname', 'lastName', 'firstName', 'action'];
   barmenData: MatTableDataSource<Barman>;
+  barmen: Barman[];
+  activeFilter: Boolean;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this 'Boolean' wrapper object with primitive type 'boolean'. ([typescript:S1533](https://sonarcloud.io/organizations/k-fet/rules#rule_key=typescript%3AS1533))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=k-app&pullRequest=302&issues=AWY86Y-S_XXc3YMreQMS&open=AWY86Y-S_XXc3YMreQMS)

[ISSUE_KEY]: AWY86Y-S_XXc3YMreQMS